runtime/oti/vm_api.h

```diff
@@ -357,6 +357,19 @@ peekClassHashTable(J9VMThread* currentThread, J9ClassLoader* classLoader, U_8* c
 J9Class*
 internalCreateArrayClass(J9VMThread* vmThread, J9ROMArrayClass* romClass, J9Class* elementClass);
 
+/**
+ * @brief Create a new J9Class to represent an array of elementClass.
+ *
+ * @param vmThread current vm thread
+ * @param romClass the ROM class associated with new RAM class. For arrays
+ * this is always the object array ROM class "[L".
```

Comment:
* "vm" should be capitalized, see line 376

* `@param` clauses don't normally end with a period, suggest parentheses:
```
 * @param romClass the ROM class associated with new RAM class (for arrays
 * this is always the object array ROM class "[L")
```

Please correct "null restricted" to "null-restricted" throughout this file and others changed by this pull request.